project-keeper-cli/src/main/java/com/exasol/projectkeeper/cli/ProjectKeeperLauncher.java

```diff
@@ -24,12 +24,18 @@ public class ProjectKeeperLauncher {
     private final Path currentWorkingDir;
 
     ProjectKeeperLauncher(final Path currentWorkingDir) {
-        this.currentWorkingDir = currentWorkingDir.toAbsolutePath();
+        try {
+            this.currentWorkingDir = currentWorkingDir.toAbsolutePath().toRealPath();
```

Comment:
According to the documentation (https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Path.html#toRealPath(java.nio.file.LinkOption...) this can be shortened:

```suggestion
            this.currentWorkingDir = currentWorkingDir.toRealPath();
```